docs/can-guides/experiment/setting-up-canjs.md

```diff
@@ -10,246 +10,360 @@ You can download CanJS from npm or a CDN. We recommend using npm. If you don’t
 ```
 
-Next, create a `main` module for your application. Import [can-define/map/map] and your template to say “Hello World”:
+When the server starts, it’ll tell you the addresses you can open in your
+browser to see your project. They will be similar to [http://localhost:8080/].
 
-```js
-// main.js
-import DefineMap from "can-define/map/map";
-import template from "./main.stache!";
+Next, you can choose to use [StealJS](#StealJS), [webpack](#webpack), or
+[Browserify](#Browserify) to load your project.
 
-const data = new DefineMap({message: "Hello World"});
+### StealJS
 
-document.body.appendChild(template(data));
-```
+> You can skip these instructions by
+> [cloning this example repo on GitHub](https://github.com/canjs/stealjs-example).
 
-Finally, create a page that loads `steal.js` and specifies `main` as the main module:
+Install [StealJS](https://stealjs.com) and the [steal-stache] plugin from npm:
 
-```html
-<html>
-  <body>
-    <script src="./node_modules/steal/steal.js" data-main="main"></script>
-  </body>
-</html>
+```
+npm install steal steal-stache --save
 ```
 
-StealJS supports “modlet” module names that end with `/`.  This means that the above could
-also be written like:
+Next, add the following [steal configuration](https://stealjs.com/docs/StealJS.configuration.html)
+to your `package.json`:
 
-```js
-// main.js
-import DefineMap from "can-define/map/";
-import template from "./main.stache!";
+```
+{
+  "name": "my-canjs-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "http-server -c-1",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "steal": {
+    "plugins": ["steal-stache"]
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "can-component": "^4.0.0",
+    "steal": "^1.11.0",
+    "steal-stache": "^4.1.0"
+  },
+  "devDependencies": {
+    "http-server": "^0.11.0"
+  }
+}
+```
+@highlight 10-12,only
 
-const data = new DefineMap({message: "Hello World"});
+Next, create an `app.stache` template for your app:
 
-document.body.appendChild(template(data));
+```html
+{{! app.stache }}
+<h1>{{message}}</h1>
 ```
-@highlight 2-2
 
-Besides ES6 modules, StealJS supports AMD and CommonJS.  You could also write `main.js` like:
+Next, create an `index.js` module for your application. Import [can-component] and
+your template to say “Hello World”:
 
 ```js
-// main.js
-import DefineMap from "can-define/map/map";
-import template from "./main.stache!";
+// index.js
+import Component from "can-component";
+import template from "./app.stache";
+
+Component.extend({
+  tag: "my-app",
+  view: template,
+  ViewModel: {
+    message: {
+      default: "Hello World"
+    }
+  }
+});
+```
 
-const data = new DefineMap({message: "Hello World"});
+Finally, create an `index.html` page that loads `steal.js` and includes your
+`<my-app>` component:
 
-document.body.appendChild(template(data));
+```html
+<!-- index.html -->
```

Comment:
I don't know if we care, but putting this comment before the `doctype` will put IE in quirks mode.